selfdrive/ui/translations/main_th.ts

```diff
@@ -0,0 +1,1303 @@
+        <location filename="../qt/widgets/drive_stats.cc" line="39"/>
+        <source>Hours</source>
+        <translation>ชั่วโมง</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/drive_stats.cc" line="44"/>
+        <source>ALL TIME</source>
+        <translation>ทั้งหมด</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/drive_stats.cc" line="46"/>
+        <source>PAST WEEK</source>
+        <translation>สัปดาห์ที่ผ่านมา</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/drive_stats.h" line="15"/>
+        <source>KM</source>
+        <translation>ก.ม.</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/drive_stats.h" line="15"/>
+        <source>Miles</source>
+        <translation>ไมล์</translation>
+    </message>
+</context>
+<context>
+    <name>DriverViewScene</name>
+    <message>
+        <location filename="../qt/offroad/driverview.cc" line="55"/>
+        <source>camera starting</source>
+        <translation>กำลังเปิดกล้อง</translation>
+    </message>
+</context>
+<context>
+    <name>InputDialog</name>
+    <message>
+        <location filename="../qt/widgets/input.cc" line="71"/>
+        <source>Cancel</source>
+        <translation>ยกเลิก</translation>
+    </message>
+    <message numerus="yes">
+        <location filename="../qt/widgets/input.cc" line="168"/>
+        <source>Need at least %n character(s)!</source>
+        <translation>
+            <numerusform>ต้องการอย่างน้อย %1 ตัวอักษร!</numerusform>
+        </translation>
+    </message>
+</context>
+<context>
+    <name>Installer</name>
+    <message>
+        <location filename="../installer/installer.cc" line="56"/>
+        <source>Installing...</source>
+        <translation>กำลังติดตั้ง...</translation>
+    </message>
+    <message>
+        <location filename="../installer/installer.cc" line="144"/>
+        <source>Receiving objects: </source>
+        <translation>กำลังรับข้อมูล: </translation>
+    </message>
+    <message>
+        <location filename="../installer/installer.cc" line="145"/>
+        <source>Resolving deltas: </source>
+        <translation>การแก้ไขเดลต้า: </translation>
+    </message>
+    <message>
+        <location filename="../installer/installer.cc" line="146"/>
+        <source>Updating files: </source>
+        <translation>กำลังอัปเดตไฟล์: </translation>
+    </message>
+</context>
+<context>
+    <name>MapETA</name>
+    <message>
+        <location filename="../qt/maps/map.cc" line="621"/>
+        <source>eta</source>
+        <translation>eta</translation>
+    </message>
+    <message>
+        <location filename="../qt/maps/map.cc" line="631"/>
+        <source>min</source>
+        <translation>นาที</translation>
+    </message>
+    <message>
+        <location filename="../qt/maps/map.cc" line="635"/>
+        <source>hr</source>
+        <translation>ช.ม.</translation>
```

Comment:
Google Translate says this means "watch," does this mean "hour"?

Reply:
watch in Thai is ชม (no dot)

hour and hours in Thai is ชั่วโมง (no plural)
ช.ม. (with dot between letter) is short for ชั่วโมง, like hr. in english.